reflex/components/radix/themes/components/context_menu.py

```diff
@@ -45,37 +63,64 @@ class ContextMenuContent(RadixThemesComponent):
 
     tag = "ContextMenu.Content"
 
-    # Button size "1" - "4"
-    size: Var[Responsive[Literal["1", "2"]]]
+    # Dropdown Menu Content size "1" - "2"
+    size: Var[Responsive[LiteralSizeType]]
 
-    # Variant of button: "solid" | "soft" | "outline" | "ghost"
-    variant: Var[Literal["solid", "soft"]]
+    # Variant of Dropdown Menu Content: "solid" | "soft"
+    variant: Var[LiteralVariantType]
 
-    # Override theme color for button
+    # Override theme color for Dropdown Menu Content
     color_scheme: Var[LiteralAccentColor]
 
-    # Whether to render the button with higher contrast color against background
+    # Renders the Dropdown Menu Content in higher contrast
     high_contrast: Var[bool]
 
-    # The vertical distance in pixels from the anchor.
-    align_offset: Var[int]
+    # Change the default rendered element for the one passed as a child, merging their props and behavior. Defaults to False.
+    as_child: Var[bool]
+
+    # When True, keyboard navigation will loop from last item to first, and vice versa. Defaults to False.
+    loop: Var[bool]
+
+    # Used to force mounting when more control is needed. Useful when controlling animation with React animation libraries.
+    force_mount: Var[bool]
 
-    # When true, overrides the side and aligns preferences to prevent collisions with boundary edges.
+    # The preferred side of the trigger to render against when open. Will be reversed when collisions occur and `avoid_collisions` is enabled.The position of the tooltip. Defaults to "top".
+    side: Var[LiteralSideType]
+
+    # The distance in pixels from the trigger. Defaults to 0.
+    side_offset: Var[Union[float, int]]
+
+    # The preferred alignment against the trigger. May change when collisions occur. Defaults to "center".
+    align: Var[LiteralAlignType]
+
+    # An offset in pixels from the "start" or "end" alignment options.
+    align_offset: Var[Union[float, int]]
+
+    # When true, overrides the side and align preferences to prevent collisions with boundary edges. Defaults to True.
     avoid_collisions: Var[bool]
 
-    # Fired when the context menu is closed.
+    # The distance in pixels from the boundary edges where collision detection should occur. Accepts a number (same for all sides), or a partial padding object, for example: { "top": 20, "left": 20 }. Defaults to 0.
+    collision_padding: Var[Union[float, int, Dict[str, Union[float, int]]]]
+
+    # The sticky behavior on the align axis. "partial" will keep the content in the boundary as long as the trigger is at least partially in the boundary whilst "always" will keep the content in the boundary regardless. Defaults to "partial".
+    sticky: Var[LiteralStickyType]
+
+    # Whether to hide the content when the trigger becomes fully occluded. Defaults to False.
+    hide_when_detached: Var[bool]
+
+    # Fired when the dialog is closed.
```

Comment:
This description for `on_close_auto_focus` doesn't seem quite right

Reply:
you're right, fixed